src/ontology/mondo-edit.obo

```diff
@@ -201187,7 +201187,7 @@ synonym: "Wnd" RELATED [OMIM:277900]
 xref: DOID:893 {source="MONDO:equivalentTo"}
 xref: GARD:7893 {source="Orphanet:905"}
 xref: ICD10CM:E83.0 {source="Orphanet:905/inclusion", source="Orphanet:905/ntbt", source="Orphanet:905"}
-xref: ICD10CM:E83.01 {source="DOID:893"}
+xref: ICD10CM:E83.01 {source="MONDO:equivalentTo", source="DOID:893"}
```

Comment:
@nicolevasilevsky Should we also add the orcid of the person who reported this as a source?

Reply:
I like this idea, will do. I think it would be a good idea in general to include ORCIDs for the requester, to give them credit and for provenance.